programs/sbf/benches/bpf_loader.rs

```diff
@@ -128,14 +128,13 @@ fn bench_program_alu(bencher: &mut Bencher) {
             .unwrap();
 
     verified_executable.jit_compile().unwrap();
+    let invoke_context = &mut invoke_context;
```

Comment:
Is there a reason you removed reborrowing in my original change? IMO it looked
better - it's a compiler only thing, and allows the callers to not have to worry
about the fact that it's a macro and not a call.

Reply:
Nope, must have resolved it the wrong way. Should be fixed now.